docs/circuits.ipynb

```diff
@@ -579,6 +579,137 @@
     "Especially useful is dropping the last moment (which are often just measurements): `circuit[:-1]`, or reversing a circuit: `circuit[::-1]`.\n"
```

Comment:
`qubits = [cirq.GridQubit(x, y) for x in range(3) for y in range(3)]` can be qubits=cirq.GridQubit.square(3)

Reply:
Done.